### ISSUE TITLE
Shortcut for Recent Files not working if diagram is in fullscreen

### ISSUE THREAD

**Issue by ehmkah** [docToolchain/diagrams.net-intellij-plugin#49]
If I open a drawio file in Fullscreenmode (CTRL+12) its not possible to use Recent-Files (CTRL+E) to go an other last edited file.

IDE: Intellij 2020.3.1
Plugin version: 0.1.4
OS: Windows 10
Keybindings: default

**Comment by rdmueller** [comment]
ah, I see. btw: can't make ctrl+F12 work on my machine.
ctrl+E seems to be an intellij shortcut and when diagrams.net is in fullscreen, it captures all keystrokes and doesn't get them through to intellij.

I think, this is the code where this can be fixed:
https://github.com/docToolchain/diagrams.net-intellij-plugin/blob/main/src/webview/index.html#L325

**Comment by hediet** [comment]
Yes, they are still specific to VS Code and need to be adapted to IntelliJ shortcuts!
